recipe/build_cxx.sh

```diff
@@ -9,15 +9,7 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
   export CMAKE_ARGS="${CMAKE_ARGS} -Dgz-msgs10_PYTHON_INTERPRETER=$BUILD_PREFIX/bin/python -Dgz-msgs10_PROTOC_EXECUTABLE=$BUILD_PREFIX/bin/protoc -Dgz-msgs10_PROTO_GENERATOR_PLUGIN=$BUILD_PREFIX/bin/gz-msgs10_protoc_plugin"
 fi
 
-env
-
-# PyPy does not support embedding the interpreter, see 
-# https://github.com/conda-forge/gz-sim-feedstock/pull/26#issuecomment-1755196585
-if [[ "${python_impl}" == "pypy" ]] ; then
-  export SKIP_PYBIND11=ON
-else
-  export SKIP_PYBIND11=OFF
-fi
+export SKIP_PYBIND11=ON  # python bindings are built in build_python.sh
```

Comment:
I think we need to revert that. Bindings compilation in `build_cxx.sh` is already skipped by the modifications in `standalone_bindings.patch`. `SKIP_PYBIND11` needs to be set to `OFF` to to ensure that the support for running gz's systems (i.e. plugins) written in Python is available in the core gz-sim library.

Reply:
If you are ok I can do the required modifications.

Reply:
Yes please :)